crypto.go

```diff
@@ -398,6 +425,33 @@ func Sign(message []byte, key SigningKey) []byte {
 	return bytes[:offset]
 }
 
+// Verify a signature of message using the provided public key, for messages signed with TFSP1;ED25519;BLAKE2B
+func Verify(signature string, message string, publicSigningKey []byte) (string, error) {
```

Comment:
lets use this type please for the public signing key https://github.com/tozny/e3db-clients-go/pull/175/files#diff-69c5c48c1da9bf0e59ac32c7d0c0089fee30329073ffcc86cf7200f224f44ecfR50

Reply:
This causes a need to copy the slice later but the overhead is not massive

Reply:
That link doesn't work